Source/Atomic/IO/PackageFile.cpp

```diff
@@ -166,12 +166,18 @@ void PackageFile::Scan(Vector<String>& result, const String& pathName, const Str
     if (filterExtension.Contains('*'))
         filterExtension.Clear();
 
+    bool caseSensitive = true;
+#ifdef _WIN32
+    // On Windows ignore case in string comparisons
+    caseSensitive = false;
+#endif
+
     const StringVector& entryNames = GetEntryNames();
     for (StringVector::ConstIterator i = entryNames.Begin(); i != entryNames.End(); ++i)
     {
         String entryName = GetSanitizedPath(*i);
-        if ((filterExtension.Empty() || entryName.EndsWith(filterExtension)) &&
-            entryName.StartsWith(sanitizedPath))
+        if ((filterExtension.Empty() || entryName.EndsWith(filterExtension, caseSensitive)) &&
+            entryName.StartsWith(sanitizedPath, caseSensitive))
         {

```

Comment:
Ok, maybe need a bool parameter on the case insensitive, which Windows builds pass in?  It would be nice if .pak files were universal though, hrm.  Gotcha on the ```_WIN32``` would be good to go through and fix the inconsistencies there, no worries.